kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseGroup.java

```diff
@@ -413,16 +413,16 @@ public void addMember(String userId, String roleId, boolean active, boolean prov
 	{
 		try {
 			insertMember(userId, roleId, active, provided);
-		} catch (IllegalStateException ise) {
-			log.warn(ise.getMessage());
+		} catch (IllegalStateException | AuthzRealmLockException isearle) {
```

Comment:
Let's not call this exception `isearle` :)

Reply:
Changed to iseorarle!